Package.swift

```diff
@@ -33,6 +33,17 @@ func resolveTargets() -> [Target] {
     return baseTargets
 }
 
+
+// Only add DocC Plugin when building docs, so that clients of this library won't
+// unnecessarily also get the DocC Plugin
+let environmentVariables = ProcessInfo.processInfo.environment
+let shouldIncludeDocCPlugin = environmentVariables["INCLUDE_DOCC_PLUGIN"] == "true"
```

Comment:
comment says it all: if we don't do this, then anyone who uses our library also gets a dependency on the DocC plugin, which could be problematic if you have another package that has a different version of the dependency